packages/Button/src/Button.tsx

```diff
@@ -305,9 +305,7 @@ interface LinkButtonProps extends HTMLElementProps<'a'>, SharedButtonProps {
   href: string;
 }
 
-interface ButtonButtonProps
-  extends HTMLElementProps<'button'>,
-    SharedButtonProps {
+interface ButtonButtonProps extends SharedButtonProps, SharedButtonProps {
```

Comment:
Why is this extending `SharedButtonProps` twice now?

Reply:
Fixed that, I was messing around with  the props trying to figure out what was making ts:check fail, but it turns out it was just adding size/variant to the test files.